omod/src/main/webapp/fragments/tags.gsp

```diff
@@ -0,0 +1,58 @@
+<%
+    ui.includeJavascript("uicommons", "angular.min.js")
+    ui.includeJavascript("tag","app.js")
+    ui.includeJavascript("tag", "service/tagService.js")
+    ui.includeJavascript("uicommons", "angular.js")
+    ui.includeJavascript("uicommons", "angular-resource.min.js")
+    ui.includeJavascript("uicommons", "ngDialog/ngDialog.js")
+    ui.includeCss("uicommons", "ngDialog/ngDialog.min.css")
+
```

Comment:
You need to include:

ui.includeJavascript("uicommons", "angular-common.js")
ui.includeJavascript("uicommons", "angular-common-error.js")